lib/ImportChecker.js

```diff
@@ -27,6 +27,9 @@ class ImportChecker {
             return importPath === this.name;
         }
         if (this.regExp) {
+            if (typeof importPath !== 'string') {
```

Comment:
@AlexMost the null check for `requireArg` was merged in #31 but this issue still persists atm